tests/metrics_test.py

```diff
@@ -0,0 +1,229 @@
+# This file is part of sbi, a toolkit for simulation-based inference. sbi is licensed
```

Comment:
there are a lot of tests in this file, do we need all of them? E.g., when the c2st-function from above allows to perform c2st with different classifiers by passing different args, then one could test them more efficiently, no? 

In addition, could it be that some of these tests still try to compare the performance of the "old" MLP-based c2st with the "new" one with RFs? If so, I think these can go out because we don't need to test for this, but only analyse it once I think. would you agree?

Reply:
I reduced them to 6 by now: 3 using a randomforest (those are very fast), 3 using the MLP (those are super slow). I'm happy to ditch the latter completely. But if you guys want to keep the MLP as default, that doesn't make too much sense.